synapseutils/copy.py

```diff
@@ -104,7 +146,7 @@ def _copyRecursive(syn, entity, destinationId, mapping=None, **kwargs):
     if (isinstance(ent, Project) or isinstance(ent, Folder)) and version is not None:
         raise ValueError("Cannot specify version when copying a project of folder")
 
-    if not isinstance(ent, (Project, Folder, File, Link, Schema)):
+    if not isinstance(ent, (Project, Folder, File, Link, Schema, Entity)):
```

Comment:
No it doesn't yet because the other one doesn't have this feature.  Will have to accept this pull request first before.